README.md

```diff
@@ -16,7 +13,7 @@ This web UI is used to view metadata collected and cataloged by [Marquez](https:
 
 ### Status
 
-This project is in the early stages of development.
+This project is **deprecated.** Please migrate to [`marquez/web`](https://github.com/MarquezProject/marquez/tree/main/web).
```

Comment:
I would say, that: this *repository* is deprecated and the project has been moved to marquez/web
The project itself keeps going, it is just being moved.
Do we want to also have a last commit on main that deletes everything and keeps only a README that points to the new location?
The history would be preserved and that would make things really obvious.

Reply:
Yep, great suggestion!